hnn_core/hnn_io.py

```diff
@@ -439,7 +437,7 @@ def read_network_configuration(fname, read_drives=True):
                                         net_data['external_drives'])
     for key in external_drive_data.keys():
         _read_external_drive(net, external_drive_data[key],
-                             read_output=False, read_drives=read_drives)
+                             read_output=False)
```

Comment:
Making sure I understand, the idea is that `net.clear_drives()` updates both `net.connectivity` and `net.external_drives`? Whereas this previously only updated (rather omitted updating) the external drive

Reply:
Yep. Previously it was built into `read_external_drive` to return None if when supplied `read_drives=false` and we weren't checking and clearing the connectivity. This change just builds everything and uses the Network api's `clear_drives` to remove the drive information (both from `net.external_drives` and `net.connectivity`).